.github/workflows/test.yml

```diff
@@ -39,9 +42,13 @@ jobs:
           CMAKE_DUMMY_WEBUI: true
       - run: tarantoolctl rocks remove ddl --force
 
-      - run: luacheck .
+      - name: Run luatest
+        run: luacheck .
```

Comment:
The name is 'Run luatest', but we run luacheck here.

Reply:
Fixed